<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialNetworkWindow</class>
 <widget class="QMainWindow" name="SocialNetworkWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SocialNetworkWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>961</width>
      <height>481</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>147</red>
          <green>158</green>
          <blue>195</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Accent">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>217</red>
          <green>217</green>
          <blue>217</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>147</red>
          <green>158</green>
          <blue>195</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Accent">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Light">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Midlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Dark">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Mid">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>170</red>
          <green>170</green>
          <blue>170</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="BrightText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Shadow">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="AlternateBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipBase">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>220</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ToolTipText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="PlaceholderText">
        <brush brushstyle="SolidPattern">
         <color alpha="127">
          <red>127</red>
          <green>127</green>
          <blue>127</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Accent">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="loginTextEdit">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>210</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="loginLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter your name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="loginFailedLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>310</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="createAccountButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>370</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Account</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QTextEdit" name="createName">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="createYear">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="createZip">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="enterNameLabel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>130</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterNameLabel2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>170</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FirstName LastName</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterYearLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter birth year:</string>
      </property>
     </widget>
     <widget class="QLabel" name="enterZipLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter zip code:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="officialCreateAccountButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>350</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Account</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="semiPrivateButton">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>220</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Semi-Private</string>
      </property>
     </widget>
     <widget class="QPushButton" name="publicButton">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>220</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Public</string>
      </property>
     </widget>
     <widget class="QPushButton" name="privateButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Private</string>
      </property>
     </widget>
     <widget class="QLabel" name="privacyLevelLabel">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>160</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Select your privacy level:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="viewableLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="profileLabel">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>50</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>hi</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="friendsTableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>131</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="suggestedFriendsLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Suggested Friends</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="suggestedFriendsTableWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>280</y>
        <width>131</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="myprofileButton">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>121</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Go to My Profile</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addFriendButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Friend</string>
      </property>
     </widget>
     <widget class="QLabel" name="recentPostsLabel1">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="addSuggestedFriendLabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>380</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="searchBar">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="searchButton">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLabel" name="friendsLabel">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Friends</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeFriendButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>430</y>
        <width>111</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Friend</string>
      </property>
     </widget>
     <widget class="QPushButton" name="writePostButton">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>430</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Write Post</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="writePostsTextEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>160</y>
        <width>641</width>
        <height>131</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="publishButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>310</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Publish</string>
      </property>
     </widget>
     <widget class="QComboBox" name="postPrivacySetting">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>310</y>
        <width>103</width>
        <height>32</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Public</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Semi-Private</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Private</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="recentPostsLabel2">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="recentPostsLabel3">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>240</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="recentPostsLabel4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>300</y>
        <width>571</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="recentPostsLabel5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>370</y>
        <width>571</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="likeButton1">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>130</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
     <widget class="QPushButton" name="likeButton2">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>190</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
     <widget class="QPushButton" name="likeButton3">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>250</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
     <widget class="QPushButton" name="likeButton4">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>310</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
     <widget class="QPushButton" name="likeButton5">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>380</y>
        <width>61</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Like</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
